src/http.py

```diff
@@ -0,0 +1,78 @@
+###Finding records in Sierra that do not have working links###
+
+import csv
+import requests
+from pymarc import Field,Subfield,MARCReader
+
+class Rowish:
```

Comment:
I find `namedtuples` very useful when dealing with our data.
See here: https://docs.python.org/3/library/collections.html#collections.namedtuple
especially see the example how neatly `namedtuples` can be used with tabular data (csv)

Reply:
Said that, this is the first `class` in Python I saw from you and it happened very quickly. Impressed~